frontend/src/admin/admin.tsx

```diff
@@ -53,8 +53,15 @@ export const Admin: FunctionComponent = () => {
                                 ))}
                             </>
                         )}/>
+
                         <Column field="createdAtToString" body={(survey: Survey ) => formatDatetime(survey.createdAt.toString())} header="Opgestuurd op" sortable/>
-                        <Column field="createdBy" body={(survey: Survey ) => survey.createdBy?.note} header="Aangemaakt door" sortable/>
+                        <Column field="createdByToString" header="Aangemaakt door" sortable filter />
+                        <Column field="active" header="Actief" sortable
+                                body={(survey: Survey) => <SurveyActiveCheckbox
```

Comment:
It is a bit confusing the display status with the action to change the status, and the icons, instead of showing only 'green' for active and 'red' for inactive. A new issue is created to improve the UI/UX. https://github.com/Zenmo/zero-web/issues/167

Reply:
Ack.